scipy/stats/_sensitivity_analysis.py

```diff
@@ -81,11 +81,12 @@ def sample_A_B(
     A, B = A_B[:, :d], A_B[:, d:]
 
     for d_, dist in enumerate(dists):
-        dist_rng = NumericalInversePolynomial(
-            dist, random_state=random_state
-        )
-        A[:, d_] = dist_rng.ppf(A[:, d_])
-        B[:, d_] = dist_rng.ppf(B[:, d_])
+        if not hasattr(dist, 'ppf'):
+            dist = NumericalInversePolynomial(
+                dist, random_state=random_state
+            )
```

Comment:
Hmm, this wasn't what I meant. I'd rather leave it as it was for now.
```suggestion
        dist = NumericalInversePolynomial(
            dist, random_state=random_state
        )
```
The intent was to evaluate which is faster - built-in `ppf` or `NumericalInversePolynomial.ppf` - for each distribution in a followup.